src/libraries/Microsoft.AspNetCore.Internal.Transport/src/Microsoft.AspNetCore.Internal.Transport.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <!-- These are wrapper project files for packaging.-->
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0</TargetFrameworks>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <!-- Avoid trying to binplace non existent documentation file.-->
+    <EnableBinPlacing>false</EnableBinPlacing>
```

Comment:
EnableBinplacing should be disabled by default for `UsingMicrosoftNoTargetsSdk` projects: https://github.com/microsoft/MSBuildSdks/blob/1a1e144dd2d5116a280b3f87e1d4ff49c30619b4/src/NoTargets/Sdk/Sdk.props#L10.